variant/translators/translator.py

```diff
@@ -0,0 +1,19 @@
+"""Module for translation."""
+from abc import ABC, abstractmethod
+from variant.schemas.ga4gh_vrs import Allele
+from variant.schemas.validation_response_schema import ValidationResult
+from variant.schemas.classification_response_schema import ClassificationType
+
+
+class Translator(ABC):
+    """The translation class."""
+
+    @abstractmethod
+    def translate(self, res: ValidationResult) -> Allele:
+        """Translate a validation result to a VRS representation."""
+        pass
```

Comment:
these should probably raise NotImplementedErrors rather than pass

https://docs.python.org/3/library/exceptions.html#NotImplementedError

Reply:
(ditto to the other ABCs as well)

Reply:
You're right